pages/login.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://ui-ic.netlify.app/documentation/styles.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
```

Comment:
since these stylesheets will be needed in every page, then its better to import them in a single CSS file and import that file in the CSS file for your page (like the `login.css` here) , this way you won't need to import them everywhere and your HTML would be clean

Reply:
Done